### PR TITLE
(chore) O3-2989 service queues - refactor <PatientSearch> to not have…

### PR DESCRIPTION
**Pull request by chibongho**
… things unrelated to patient serach

## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary

In service queues app, the `<PatientSearch>` component is an overlay workspace for user to serach for a patient and add them to the queue. However, the component is overloaded with other use cases of overlay workspaces. Refactor that out of the component.

There are really 3 types of workspace overlays used in the queues app: PatientSearch form, New Queue Service form, and New Queue Room form. There is additional logic in PatientSearch to show different forms depending on whether the user has an active visit or not. The logic is also overloaded with logic to display the other 2 forms. This PR factors out those logic.

### CODE REVIEW

File: packages/esm-appointments-app/translations/en.json [hunk 1]
```diff
@@ -168,6 +167,5 @@
   "visitType": "Visit Type",
   "vitals": "Vitals",
   "week": "Week",
-  "weekly": "Weekly",
   "yes": "Yes"
 }
```
Comment (mseaton) [inline packages/esm-appointments-app/translations/en.json:171]: These changes seem unrelated?  Maybe you need to update your branch?

**Review by mseaton — APPROVED** [review]
Looks good to me @chibongho !  Thanks for simplifying this!

**Comment by ojwanganto** [comment]
Thanks, @chibongho. Looks good to me.

File: packages/esm-appointments-app/translations/en.json (hunk 1 above)
Comment (chibongho) [reply packages/esm-appointments-app/translations/en.json:171]: This is autogenerated from the git push hooks. My guess is that some previous commit in the main branch neglected to include this change.
